torch/_dynamo/output_graph.py

```diff
@@ -567,7 +567,10 @@ def subtracer(self, source_target, prior_tracer):
                 prior_tracer
                 if prior_tracer
                 else SubgraphTracer(
-                    self, parent=self.current_tracer, source_target=source_target
+                    self,
```

Comment:
The other thing I'm curious about is: how important is it to get the control flow operators working without Dynamo? I know that we want to turn non-strict export on by default, does that involve also figuring out how to eliminate Dynamo from the control flow ops?

Reply:
It seems to me OK to turn on dynamo for control flow operators in non-strict export. Non-strict makes exporting a large set of models easier because they don't need to deal with coverage issues of dynamo. However, when people need control flow operators, they always need to do some manual rewrite. So it seems ok to turn on dynamo here if we can provide a good error message. Since control flow operators operate on a local region of a program, I feel the complexity is less. Besides, an auto-converter would need dynamo if we really want to scale the usage of control flow operators.